### PR TITLE
[HHCTRL.OCX] Add help viewer window icon and default title

### PR DESCRIPTION
**Pull request by TAN-Gaming**
## Purpose

- Add help viewer window icon and default title

Before:
VirtualBox_ReactOS_06_06_2022_14_19_09

After:
VirtualBox_ReactOS_06_06_2022_14_25_32

### CODE REVIEW

File: dll/win32/hhctrl.ocx/chm.c [hunk 1]
```diff
@@ -459,7 +460,7 @@ BOOL LoadWinTypeFromCHM(HHInfo *info)
     /* merge the new data with any pre-existing HH_WINTYPE structure */
     MergeChmProperties(&wintype, info, FALSE);
     if (!info->WinType.pszCaption)
-        info->WinType.pszCaption = info->stringsW.pszCaption = strdupW(info->pCHMInfo->defTitle ? info->pCHMInfo->defTitle : empty);
+        info->WinType.pszCaption = info->stringsW.pszCaption = strdupW(info->pCHMInfo->defTitle ? info->pCHMInfo->defTitle : def_titleW);
```
Comment (learn-more) [inline dll/win32/hhctrl.ocx/chm.c:463]: Can just put `L"Help"` here directly

File: dll/win32/hhctrl.ocx/help.c [hunk 2]
```diff
@@ -1620,12 +1620,12 @@ static BOOL HH_CreateHelpWindow(HHInfo *info)
...
     wcex.cbWndExtra     = sizeof(LONG_PTR);
     wcex.hInstance      = hhctrl_hinstance;
-    wcex.hIcon          = LoadIconW(NULL, (LPCWSTR)IDI_APPLICATION);
+    wcex.hIcon          = LoadIconW(hhctrl_hinstance, MAKEINTRESOURCEW(IDI_HHICON));
     wcex.hCursor        = LoadCursorW(NULL, (LPCWSTR)IDC_ARROW);
     wcex.hbrBackground  = (HBRUSH)(COLOR_MENU + 1);
     wcex.lpszMenuName   = NULL;
     wcex.lpszClassName  = windowClassW;
-    wcex.hIconSm        = LoadIconW(NULL, (LPCWSTR)IDI_APPLICATION);
+    wcex.hIconSm        = NULL;
```
Comment (learn-more) [inline dll/win32/hhctrl.ocx/help.c:1628]: Why don't you set the small icon?

Comment (TAN-Gaming) [reply dll/win32/hhctrl.ocx/help.c:1628]: > If this member is NULL, the system searches the icon resource specified by the hIcon member for an icon of the appropriate size to use as the small icon.

[https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-wndclassexw](https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-wndclassexw)

File: dll/win32/hhctrl.ocx/chm.c (hunk 1 above)
Comment (TAN-Gaming) [reply dll/win32/hhctrl.ocx/chm.c:463]: I just follow the original coding style.

Comment (TAN-Gaming) [reply dll/win32/hhctrl.ocx/chm.c:463]: I just follow the original coding style.

Comment (TAN-Gaming) [reply dll/win32/hhctrl.ocx/chm.c:463]: I use the original coding style, for readability.

Comment (gonzoMD) [reply dll/win32/hhctrl.ocx/chm.c:463]: Why don't you add the string to the language resource files?

**Review by learn-more — APPROVED** [review]

File: dll/win32/hhctrl.ocx/chm.c (hunk 1 above)
Comment (TAN-Gaming) [reply dll/win32/hhctrl.ocx/chm.c:463]: > Why don't you add the string to the language resource files?

That's a good idea.

Comment (TAN-Gaming) [reply dll/win32/hhctrl.ocx/chm.c]: > Why don't you add the string to the language resource files?

Oh, I forgot about the translations. 😄

**Review by binarymaster — COMMENTED** [review]
But... this module is WineSync'ed: https://github.com/reactos/reactos/blob/master/media/doc/WINESYNC.txt#L75

**Comment by TAN-Gaming** [comment]
> But... this module is WineSync'ed: https://github.com/reactos/reactos/blob/master/media/doc/WINESYNC.txt#L75

Can I use `__REACTOS__` in .rc files?

**Pull request merged**